src/NServiceBus.Core/Routing/FileBasedDynamicRouting/endpoints.xsd

```diff
@@ -0,0 +1,21 @@
+﻿<xs:schema attributeFormDefault="unqualified" elementFormDefault="qualified" xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="endpoints">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element maxOccurs="unbounded" name="endpoint">
+          <xs:complexType>
+            <xs:sequence>
+              <xs:element maxOccurs="unbounded" name="instance">
```

Comment:
would probably make sense to explicitly state that it requires at least one element by specifying `minOccurs="1"` instead of the implicit default value?